src/main/java/uk/nhs/digital/emailadapter/service/StagedStopwatch.java

```diff
@@ -20,12 +19,12 @@ private String getTimeUntilNow(Instant now) {
         .minus(seconds, ChronoUnit.SECONDS)
         .until(now, ChronoUnit.MILLIS);
 
-    return String.format("%d.%ds", seconds, millis);
+    return String.format("%d.%ds {} {} ", seconds, millis);
   }
 
   public void finishStage(String stageName) {
     Instant now = Instant.now();
-    logger.info(String.format("finished %s in %s\n", stageName, getTimeUntilNow(now)));
+    log.info(String.format("finished %s in %s\n {} {} ", stageName, getTimeUntilNow(now)));
```

Comment:
should read like below: why do you have String.format?

log.info("finished {} {} ", stageName, getTimeUntilNow(now));